lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -950,6 +953,9 @@ void OpenCLFunction::execute() {
     }
 
     if (auto *CG = dyn_cast<ConvolutionGradInst>(&I)) {
+      assert(CG->getGroup() == 1 &&
+        "Group Convolution is not supported by OpenCL backend.");
```

Comment:
Should we use GLOW_ASSERTs instead?

Reply:
No, I think we usually keep going if dimensions don't match or something minor like this is wrong. We want as less checks as possible in release mode, right?